package.json

```diff
@@ -7,7 +7,7 @@
   "description": "A blazing fast unit test framework powered by Vite",
   "scripts": {
     "ci": "ni && nr typecheck && nr lint && nr build && nr test:all",
-    "build": "pnpm -r --filter='./packages/**' run build && pnpm -C ./packages/browser run copy",
+    "build": "pnpm -r --filter='./packages/**' --filter !browser run build && pnpm --filter browser run build",
```

Comment:
It's already the last one, because it has a UI in dependencies, so this should not be needed

Reply:
Great, I'll remove this second step. So it's `pnpm` who knows that `packages/browser` needs to wait for `packages/ui` to finish?

Reply:
Yes, it is ordered automatically based on what depends on what